project-generated-code-backend/Backend/Repository/Generic/IRoomRenovationRepository.cs

```diff
@@ -0,0 +1,12 @@
+﻿using HealthClinicBackend.Backend.Model.Schedule;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace HealthClinicBackend.Backend.Repository.Generic
+{
+    interface IRoomRenovationRepository : IGenericRepository<RoomRenovation>
+    {
+        List<RoomRenovation> GetByRoomRenovationTimeInterval(string roomRenovationSerialNumber);
```

Comment:
GetByTimeInterval by convention we don't put the entity name in the method because it's already in the class name